158 replace term survey (#204)
DE-RSE/2023_paper-RSE-groups: paper.tex

Instead, we highlight central RSE units.
Motivation for their existence is discussed, underpinned by working examples both in neighbouring fields as well as outside of Germany.
The heart of this paper is a vision of a central RSE unit, its structure, and the definition of nine core support modules such a unit may provide.

> We collected feedback from existing RSE departments and found that there is considerable diversity within the module distribution, even for such a small group.

Reviewer: ```suggestion
We collected feedback from existing RSE departments and found that there is considerable diversity within the module distribution, even in such a small sample.
```
